Sources/ApertureCLI/notifications.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+
+final class ApertureNotification {
+  private var notification: Notification
+  var isAnswered = false
+
+  init(_ notification: Notification) {
+    self.notification = notification
+  }
+
+  func getField<T>(_ name: String) -> T? {
+    return notification.userInfo?[name] as? T
+  }
+
+  func getData() -> String? {
+    return getField("data")
+  }
+
+  func answer() {
+    answer(nil)
+  }
+
+  func answer(_ data: Any?) {
```

Comment:
```suggestion
  func answer(_ data: Any? = nil) {
```

and you can drop `func answer()`